launch/launch/actions/include_launch_description.py

```diff
@@ -125,6 +128,20 @@ def get_sub_entities(self):
         ret = self.__launch_description_source.try_get_launch_description_without_context()
         return [ret] if ret is not None else []
 
+    def _try_get_arguments_names_without_context(self):
+        try:
+            context = LaunchContext()
+            return [
+                perform_substitutions(context, normalize_to_list_of_substitutions(arg_name))
+                for arg_name, arg_value in self.__launch_arguments
+            ]
+        except Exception:
+            self.__logger.debug(
+                'Failed to get launch arguments names for launch description: '
+                f'{self.__launch_description_source.location}'
```

Comment:
Is it possible to give a bit more context here? maybe just `str()` the exception at the end?

Reply:
Sounds good, fixed in https://github.com/ros2/launch/pull/556/commits/2aa41db6636a23c34257acf4af15f1f7e3b89b86.